R/remove_empties.R

```diff
@@ -100,7 +100,7 @@ remove_empty <- function(dat, which = c("rows", "cols"), cutoff = 1, quiet = TRU
 #'   unique()
 #' @importFrom stats na.omit
 #' @family remove functions
-#' @seealso \code{\link[=remove_empty]{remove_empty()}} for removing empty
+#' @seealso [`remove_empty()`][remove_empty] for removing empty
```

Comment:
Is the second [remove_empty] needed here?

Reply:
No, it is just that before, it was a link to `remove_empty()`, but it showed, as `remove_empty`, for consistency, I agree we should change it.